doc/src/modules/physics/vector/advanced.rst

```diff
@@ -198,3 +200,44 @@ so dynamic symbols created before or after will print the same way.
 Also note that ``Vector``'s ``.dt`` method uses the ``._t`` attribute of
 ``dynamicsymbols``, along with a number of other important functions and
 methods. Don't mix and match symbols representing time.
+
+
+
+Solving Vector Equations
+---------------
+Solving for unknown scalar variables in a set of vector equations is a common need, but 
+SymPy's solvers do not work directly with the physics vectors, so you must first transform
+the vector equations into a set of scalar equations. The `to_matrix()` method provides
+a simple way to do this. For example:
+
+PROBLEM:
+
+A ball is thrown from origin at a velocity of 10m/s along the x-axis w.r.t frame ``A``.
+An observer ``B``'s reference frame is rotated by 30 degrees w.r.t ``A``.
+What are the x and y component values of the velocity vector of the ball as 
+observed by the observer in its frame?
```

Comment:
A few comments:
- Could you try to use the notation from [this page](https://moorepants.github.io/learn-multibody-dynamics/notation.html)
- Don't write code like `>>> v_a = 10 * A.x` before imports and definitions
- Can you make your import explicit, i.e. `from sympy import pi, solve, symbols`
- Use consistent spacing and number of backticks `
- You could rewrite it to a normal example, which is not in the form of a dynamics problem.
- Hint: build the webpage to check what you've written, see [here](https://docs.sympy.org/latest/contributing/build-docs.html) for instructions.